vivo/Dockerfile

```diff
@@ -36,10 +36,8 @@ COPY ./log4j.properties /usr/local/tomcat/webapps/VIVO/WEB-INF/classes
 
 WORKDIR /usr/local/VIVO/home/config
 
-COPY ./dockercompose.runtime.properties runtime.properties
-COPY ./example.applicationSetup.n3 applicationSetup.n3
-#COPY ./dockercompose.runtime.properties /usr/local/vivo/home/config/runtime.properties
-#COPY ./example.applicationSetup.n3 /usr/local/vivo/home/config/applicationSetup.n3
+COPY ./runtime.properties runtime.properties
```

Comment:
I don't like these namechanges anymore.  Someone else chose them for a reason & it's foolhardy for me to single-handedly change them.  Unless you strongly feel they should stay, I'll revert them to the previous filenames.

Reply:
Your call.